packages/shared/src/hooks/referral/useReferralCampaign.ts

```diff
@@ -40,12 +41,14 @@ const campaignFeatureFlagMap: Partial<
 
 const useReferralCampaign = ({
   campaignKey,
+  enabled = true,
```

Comment:
Why did we add this here?

Reply:
In order to not fire the query if the profile is not the logged in user profile https://github.com/dailydotdev/apps/blob/b7853ddca89e22739b898a322c2e3e193c11491e/packages/webapp/components/layouts/ProfileLayout/index.tsx#L79